src/main/java/ca/corefacility/bioinformatics/irida/ria/web/announcements/AnnouncementsController.java

```diff
@@ -129,19 +129,27 @@ public String getControlCentreAdminPage(final Model model) {
      *
```

Comment:
Something seems a little weird with the formatting. Do you have intellij setup to use the irida style xml?

Reply:
I setup eslint... maybe I missed something. I will check the documentation.

Reply:
In IntelliJ, imported schema from https://irida.corefacility.ca/documentation/developer/getting-started/irida-code-format.xml, highlighted comment and method and pressed Ctrl+Alt+L keys.
Please see b7e3c83ac6ef44b1e4f97b68f13ac9ada603a3c0.